README.md

```diff
@@ -27,14 +27,6 @@ import Button from '@catho/quantum/Button';
 
 You can find a list of all available components and full documentation on how to use them at our [Storybook](https://catho.github.io/quantum/)
 
-## Using with Typescript
```

Comment:
This is supposed to be removed?

Reply:
Yes! @luizjaccao said if the project was TypeScript, it'll recognize automatically